docs/UserGuide.md

```diff
@@ -184,6 +184,101 @@ Examples:
+
+Example:
+
+- `import patientRecords.json`
+
+**Guide:** 
+
+1. Place the `json` file to be imported in the same folder as the application JAR file. In the image below `data1.json` is the desired `json` file to be imported. Note that it has the file extension `.json` and assume that it adheres to the expected format and input constraints. 
+
+   <img src="images/importStep1.png"/>
+
+2. Run the application `jar` file as per usual (see <u>Quick start</u> for more details).
+
+3. Type `import FILENAME.json` (e.g. `import data1.json`) in the command box in the application. The following message will be displayed in the results box if the import is successful and the contacts will automatically be updated. 
+
+   <img src="images/importStep3.png"/>
+
+   Note that along a `data` folder containing the file `addressbook.json` will be generated in the folder as well if it was not already previously there (as seen below). Similarly to when `export` is executed, this `addressbook.json` file will reflect the latest patient records from the application. 
+
+   <img src="images/importStep3a.png"/>
+
+**Expected format:**
+
+The following is an example of a valid JSON file content. 
+
+````
+"persons" : [ {
+    "name" : "Alicia Pauline",
+    "phone" : "94351253",
+    "email" : "alice@example.com",
+    "address" : "123, Jurong West Ave 6, #08-111",
+    "age" : "24",
+    "sex" : "Female",
+    "appointments" : [ ],
+    "tags" : [ "friends" ],
+    "note" : {
+      "appointments" : [ "11/02/2023 1100", "11/03/2023 1600" ],
+      "remark" : [ ],
+      "medication" : [ ]
+    },
+    "starredStatus" : "false"
+  }, {
+    "name" : "Benson Meier",
+    "phone" : "98765432",
+    "email" : "johnd@example.com",
+    "address" : "311, Clementi Ave 2, #02-25",
+    "age" : "37",
+    "sex" : "Male",
+    "appointments" : [ ],
+    "tags" : [ "owesMoney", "friends" ],
+    "note" : {
+      "appointments" : [ ],
+      "remark" : [ ],
+      "medication" : [ ]
+    },
+    "starredStatus" : "true"
+  }] 
+````
+
+- All patient data must be enclosed with `"persons" : []`
+
+- Each patient data must be enclosed with `{}` and seperated by a comma `,`
+
+- The following is a blank template for each patient data. You may fill in the `" "` and `[ ]` with the relevant data if you wish to manually edit patient records before importing. Remember to ensure that data adheres to the constraints of MediContact (see <u>Summary of input constraints</u> for more details).
+
+  ````
+  {
+      "name" : " ",
+      "phone" : " ",
+      "email" : " ",
+      "address" : " ",
+      "age" : " ",
+      "sex" : " ",
+      "appointments" : [ ],
+      "tags" : [ ],
+      "note" : {
+        "appointments" : [ ],
+        "remark" : [ ],
+        "medication" : [ ]
+      },
+  ````
```

Comment:
I love the way you give a template in user guide! This makes it so much clearer!